### DISCUSSION TITLE
Add conventions for Prime Video

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#263] [Ideas]
[Prime Video](https://videocentral.amazon.com/home/help?topicId=GBKB422Q9GYC7DWE&ref_=avd_sup_GBKB422Q9GYC7DWE) offers Timed Text guidelines in various languages.

Perhaps 'Amazon' should be renamed to 'Prime Video'?

**Comment by URUWorks** [comment]
v1.0.1.7

**Comment by chenlung** [comment]
I don't think all of them were added.
